ecosystem_cicd_tools/validations.py

```diff
@@ -311,8 +311,15 @@ def check_version_plugins_and_update(path, plugins, version):
                             .format(file_name=file_name,
                                     version_in_plugin=version_in_plugin,
                                     version=version ))
-        if version_in_plugin != version:
+        if version_in_plugin != version and os.environ.get('CIRCLECI') is None:
```

Comment:
It looks like this condition will miss if circleci and version_in_plugin != version.